docs/guides/airnode/calling-an-airnode/index.md

```diff
@@ -315,4 +345,31 @@ your requested data decoded in `int256`
 
 > ![Making the Request](src/s7.png)
 
+## 4. Withdrawing Funds from the sponsor wallet (optional)
+
+You can withdraw funds from the sponsor wallet to your Requester contract by
+calling the `withdraw()` function.
+
+The Airnode listens for withdrawal requests and fulfills them automatically.
+Therefore, the sponsor (in this case, the Requester contract itself) should be
+able to receive their funds from their `sponsorWallet` in a few minutes notice.
```

Comment:
```suggestion
able to receive their funds from their `sponsorWallet` within a few minutes.
```

Reply:
I've changed this part entirely. Instead of having the user call `withdrawFromContract()`, I added it under the `receive()` function to send the funds to the owner automatically. Let me know if it looks fine.

Reply:
Oh nice idea 👍